docs/src/HowToGuides/Atmos/Microphysics.md

```diff
@@ -1,125 +1,277 @@
+|``\zeta^{sno}``     | coefficient in velocity(radius) for snow        | ``\frac{m^{3/4}}{s}``    | ``2^{9/4}``                           |
+|``\eta^{sno}``      | exponent in velocity(radius) for snow           | -                        | ``0.25``                              |
 
-## Terminal velocity
+where:
+ - ``\rho_{water}`` is the density of water,
+ - ``\rho_{ice}`` is the density of ice.
 
-The terminal velocity of an individual rain drop is defined by the balance between the gravitational acceleration (taking into account the density difference between water and air) and the drag force:
+``\gamma^{sno}`` corresponds to parameter ``\alpha`` in eq(16b) in
+from [Grabowski\_1998](https://journals.ametsoc.org/doi/full/10.1175/1520-0469%281998%29055%3C3283%3ATCRMOL%3E2.0.CO%3B2 ).
+
+The terminal velocity of an individual rain drop is defined by the balance between the gravitational acceleration
+(taking into account the density difference between water and air) and the drag force.
+Therefore the ``\zeta^{rai}`` is defined as
 ```math
 \begin{equation}
-v_{drop} = \left(\frac{8}{3 \, C_{drag}} \left( \frac{\rho_{water}}{\rho} -1 \right) \right)^{1/2} (g \, r)^{1/2} = v_c(\rho) \, (g \, r)^{1/2}
+\zeta^{rai} = \left(\frac{8}{3 \, C_{drag}} \left( \frac{\rho_{water}}{\rho} -1 \right) \right)^{1/2} (g)^{1/2}
 \label{eq:vdrop}
 \end{equation}
 ```
 where:
  - ``g`` is the gravitational acceleration,
  - ``C_{drag}`` is the drag coefficient,
- - ``\rho_{water}`` is the density of water,
- - ``\rho`` is the density of air.
+ - ``\rho`` is the density of air.\
 
-The mass weighted terminal velocity ``v_t`` is defined following Ogura and Takahashi 1971
+!!! note
+
+    It would be great to replace the above simple power laws with more accurate relationships.
+    For example:
+    [Khvorostyanov\_and\_Curry\_2002](https://journals.ametsoc.org/doi/pdf/10.1175/1520-0469%282002%29059%3C1872%3ATVODAC%3E2.0.CO%3B2)
+
+## Assumed particle size distributions
+
+The particle size distributions are assumed to follow Marshall-Palmer distribution
+[Marshall\_and\_Palmer\_1948](https://journals.ametsoc.org/doi/abs/10.1175/1520-0469%281948%29005%3C0165%3ATDORWS%3E2.0.CO%3B2)
+ eq. 1:
 ```math
 \begin{equation}
-v_t = \frac{F_{rain}}{RWC}
-\label{eq:vt}
+n(r) = n_{0} exp\left(- \lambda \, r \right)
 \end{equation}
 ```
 where:
- - ``F_{rain} = \int_0^\infty n(r) \, m(r) \, v_{drop}(r) \, dr`` is the vertical flux of rain drops,
- - ``RWC = \int_0^\infty n(r) \, m(r) \, dr = \rho \, q_{rai}`` is the rain water content.
+ - ``n_{0}`` and ``\lambda`` are the Marshall-Palmer distribution parameters.
 
-Integrating over the assumed Marshall-Palmer distribution results in
+The ``n_0`` for rain and ice is assumed constant.
+The ``n_0`` for snow is defined as
 ```math
 \begin{equation}
-RWC = \frac{8 \pi \, n_{0_{MP}} \, \rho_{water}}{\lambda_{MP}^4}
-\label{eq:lambda}
+n_0^{sno} = \mu^{sno} (\rho q_{sno})^{\nu^{sno}}
```

Comment:
Same as above. Introduce a scale to make quantities that are exponentiated dimensionless, so that coefficients such as `\mu^{sno}` have units of `n_0`. (Otherwise we'll run into trouble should we ever change units in the model.)

Reply:
done